displays/photo-carousel.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+/**
+ * @var $ids array IDs for the images to be displayed in the carousel
+ * @var $name string Returns the name of the current instance from the shortcode params or the current page id (limits usage to one per page)
+ */
+?>
+
+<!-- Swiper -->
+<style>
```

Comment:
Do you need the style here? Is this included in swiper.min.css? If you do that's fine - just wanted to make sure it wasn't duplicated.

Reply:
These styles are a combination of being specific to a "carousel", but also include some "WSU branded" overrides. Also, these styles only help the "tile" view we're going for, so I thought I made sense to put them here and then we can know to move them into the block once we get there.